GL-js-hw-07/task04-07.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+// task 04
+const counterValue = document.querySelector('#value');
```

Comment:
Всі елементи, у яких є **id**, рекомендую знаходити ось так:
```suggestion
const counterValue = document.getElementById('value');
```